frontend/src/index.js

```diff
@@ -11,6 +11,8 @@ ReactDOM.render(
   document.getElementById('root')
 );
 //test error
+// comment PR
+// 18/08/2021 22:16
```

Comment:
It's fine ... but we need it in yyyy/mm/dd format thx!

Reply:
Yes sure